src/app/containers/SignupContainer/SignupContainer.js

```diff
@@ -157,7 +162,8 @@ const SignupContainer = ({ history, onLogin }) => {
                             <VerificationStep
                                 model={model}
                                 allowedMethods={signupAvailability.allowedMethods}
-                                onVerificationDone={handleVerificationDone}
+                                onVerify={(...rest) => withLoading(handleVerification(...rest))}
```

Comment:
```suggestion
                                onVerify={(data) => withLoading(handleVerification(data))}
```

Reply:
handle payment done requires two parameters: `(model, paymentParameters)` `...rest` seems like a good way to tell that this method doesn't care about params.

Reply:
same with handle verifications: `(model, code, params)`